### ISSUE TITLE
Redis-server config breaks salt and apt

### ISSUE THREAD

**Issue by aranhoide** [getlantern/lantern_aws#174]
A hack to replace the SysVinit setup with an upstart-based one renders the package uninstallable.  `dpkg --configure -a` won't work.  I don't know what that'd do to unattended upgrades, or what other ramifications this may have.

While redis servers should be launched seldom enough that fixing this manually every time would be viable, the problem would remain that any future highstate calls would (i) fail and (ii) break apt in these machines.  Highstate from master should never break any systems.  And if these config servers aren't going to be salt-managed, their configuration doesn't belong in this repository.

I only merged the relevant branch with the understanding that this would get fixed.

**Comment by myleshorton** [comment]
Is this still valid @oxtoacart and @aranhoide?

**Comment by oxtoacart** [comment]
@myleshorton Yes, but we've documented the manual workaround in the README.  Since we don't launch redis servers very often, I think we can live with that for now.

**Comment by oxtoacart** [comment]
Oh, I should note that the failure only happens right after launching.  Subsequently applying updates works smoothly.
